src/styles/ProductListing.css

```diff
@@ -0,0 +1,52 @@
+.filter-container {
+	height: 60rem;
+	background-color: var(--color-pureWhite);
+}
+
+.filter-categories {
+	padding: 1em;
+}
```

Comment:
You have used padding in most places, use utils for this. Just like you used `pad-normal-top`.
Just one suggestion in the naming convention: instead of using complete words like normal large use `sm, md, lg`. Names will be shorter and easier to use